internal/titles/bf4.go

```diff
@@ -35,8 +33,8 @@ var Bf4 = domain.GameTitle{
 		},
 	},
 	URLValidator: internal.MakePatternURLValidator(internal.Frostbite3GameIdPattern),
-	CmdBuilder:   internal.MakeOriginCmdBuilder("1007968", "1011575", "1011576", "1011577", "1010268", "1010269", "1010270", "1010271", "1010958", "1010959", "1010960", "1010961", "1007077", "1016751", "1016757", "1016754", "1015365", "1015364", "1015363", "1015362"),
+	CmdBuilder:   internal.OriginCmdBuilder{},
 	HookHandlers: []game_launcher.HookHandler{
-		internal.MakeKillProcessHookHandler(false, bf4Exe), // Launcher config executable name will be "Origin.exe", which we don't want to kill
+		internal.MakeKillProcessHookHandler(false, bf4Exe),
```

Comment:
I believe the first parameter here should be true with the second parameter removed (see comment on `bf1.go`).